interface/patient_file/summary/demographics.php

```diff
@@ -851,6 +851,12 @@ function setMyPatient() {
             border-radius: 0;
         }
 
+        <?php if (!$GLOBALS['right_justify_labels_demographics']) { ?> 
+        #DEM .label_custom {
+            text-align: left !important;
+        }
+
+        <?php } ?>
```

Comment:
I'd remove all these styles and check my PR

Reply:
@sjpadgett
I know that you want to decide on a standard, but some people like one way, other people another.  Giving an option is easy.  Why not have an option?  However, it needs to be fixed to also include insurance and perhaps other areas.  You also see that in the code where I inserted this, someone else did the say thing for something else.  Will you remove that also.

Reply:
You can leave what is already there however, you may be able to get by with just BS style. As for the global well that's for Brady because I don't mind too much.
Try to remember though that we are going to refactor a lot of this soon. I almost did it my last PR...

Reply:
hi @psoas , Can live with a new global (consider it your one "get yourself out of globals jail" card :) ) as long as don't get all that empty left whitespace I noted in the above screenshots.
Also, if you have new code changes, then would rec pushing it to this PR.